client/src/pages/Login.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import {
+    withStyles,
+    Button
+} from '@material-ui/core';
+import {Link} from 'react-router-dom';
+import validator from 'validator';
+
+import {CustomOutlinedInput} from '../components/Inputs';
+import CenteredPageHeader from '../components/CenteredPageHeader';
+import FormValidator from '../helpers/form-validation';
+
+const styles = {
+    pageContent: {
+        // Alignment
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        //Spacing & Page width
+        margin: '0 auto',
+        padding: '40px 30px',
+        maxWidth: 500
+    },
+    form: {
+        // Alignment
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        // Form sizing
+        width: '100%',
+    },
+    lastInput: {
+        // Last input has spacing to separate from bottom
+        // according to specs.
+        marginBottom: 70
+    }
+};
+
+class Login extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.validators = FormValidator([
+            {
+                field: 'email',
```

Comment:
Just wanted to make sure that we are logging in with email vs username vs both. In backend I have just username at the moment.

Reply:
I think it is email, as by the ticket for sign up page says name, email and password, but I can change that when I do integration